Bottom: fd511c34e4388ff28a0a2cb136027f8f727575ae
Top:    80fd0b0f8660c8336c295cbc2a24c2b44bdc7f73
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-09-19 12:52:07 -0500

Refresh of add-haproxy-role

---
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index 7670085f4..fea7d1b67 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -53,11 +53,11 @@
 
 # *****************************************************************************
 # Update or make the image.
-- name: Checkout image repo
-  git:
-    repo: '{{ svn_image_repo }}'
-    version: master
-    dest: '{{ docker_projects_dir }}/docker-svn'
+# - name: Checkout image repo
+#   git:
+#     repo: '{{ svn_image_repo }}'
+#     version: master
+#     dest: '{{ docker_projects_dir }}/docker-svn'
 
 - name: build image
   docker_image:
diff --git a/roles/nexus3/defaults/main.yml b/roles/nexus3/defaults/main.yml
index ddc4d7015..e891ad741 100644
--- a/roles/nexus3/defaults/main.yml
+++ b/roles/nexus3/defaults/main.yml
@@ -17,11 +17,9 @@ nexus_container_name: 'nexus3'
 nexus_image_tag: 'nexus:{{ nexus_version}}'
 nexus_backup_volume: nexus-backup
 nexus_data_volume: nexus-data
-nexus_networks: 
+nexus_networks:
   - name: nexus3
-nexus_port_args:
-  - "8081"
-  - "8443"
+nexus_port_args: []
 nexus_volumes:
   - '{{ nexus_data_volume }}:{{ nexus_data_dir }}:z'
   - '{{ nexus_backup_volume }}:{{ nexus_backup_dir }}:z'
